services/vision/mlvision/detector.go

```diff
@@ -454,3 +151,27 @@ func createDetectionFilter(minConf float64, labelMap map[string]float64) objectd
 	}
 	return nil
 }
+
+// GetIndex returns the index of an int in an array of ints
+// Will return -1 if it's not there.
+func GetIndex(s []int, num int) int {
+	for i, v := range s {
+		if v == num {
+			return i
+		}
+	}
+	return -1
+}
```

Comment:
This is repeated from the new ml/ml.go function GetIndex, would you like to use that one from ml?

Reply:
Yes! all set